source/common/common/utility.h

```diff
@@ -709,7 +707,7 @@ template <class Value> class TrieLookupTable {
       children_[child_index - min_child_] = std::move(node);
     }
 
-  private:
+    Value value_{};
```

Comment:
According to the style guide, members should be private. Can we use `friend` or accessors to expose the innards?

Reply:
Made it a struct instead. Seems daft to have a private type need to friend its parent.